TLM/TLM/State/Options.cs

```diff
@@ -25,6 +25,14 @@ public class Options : MonoBehaviour {
         // private static UITextField pathCostMultiplicator2Field = null;
 #endif
 
+        [Flags]
+        public enum PersistTo {
```

Comment:
Does this have to be flags? Why not 2 bools for Global and for Savegame?

Reply:
I tried that at first, but then encountered `None` type options (eg. the `Despawner` stuff on maintenance tab which never gets persisted). Then I switched to basic `enum` but that made some of the checks a bit cumbersome, so I transitioned to `[Flags]`.

Reply:
An alternative would possibly be some sort of "persister" object or function that could be passed in.

For example, if a setting is both savegame and global config, the persister could read/write to both locations and additionally trigger a write of the global config. I'm not sure how to implement that though - eg. how do I get a field on the `Instance.Main` (or any other of the global config classes)?